__test__/helper.js

```diff
@@ -0,0 +1,35 @@
+import webdriver from 'selenium-webdriver'
+import dotenv from 'dotenv';
+dotenv.config({path: '../.env.local'})
```

Comment:
@captainamiedi is there any reason to keep this? It did not work for me. That's why I moved it to jest.setup.js.